### PR TITLE
Add LeagueSport schema

### LINKED ISSUE #278: Need to keep track of Sports for each Fantasy League
Need to use this join table to build rosters and limit available players in drop downs for 2018.

### PR DESCRIPTION
**Pull request by axelclark**
* Add LeagueSport schema
* Used to keep track of the sports for each Fantasy League
* Will enable different sports over the years
* Closes #278

### CODE REVIEW

File: web/models/league_sport.ex [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    belongs_to :fantasy_league, Ex338.FantasyLeague
+    belongs_to :sports_league, Ex338.SportsLeague
+
+    timestamps()
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`.
+  """
+  def changeset(struct, params \\ %{}) do
```
Comment (credoci) [inline web/models/league_sport.ex]: Parameter `struct` has same name as the `Kernel.struct` function.

File: web/models/league_sport.ex [hunk 2]
```diff
@@ -0,0 +1,19 @@
+defmodule Ex338.LeagueSport do
```
Comment (credoci) [inline web/models/league_sport.ex]: Modules should have a @moduledoc tag.

File: web/admin/league_sport.ex [hunk 3]
```diff
@@ -0,0 +1,7 @@
+defmodule Ex338.ExAdmin.LeagueSport do
```
Comment (credoci) [inline web/admin/league_sport.ex]: Modules should have a @moduledoc tag.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 16 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/280.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 14 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/280.

**Pull request merged**